tests/python_client/testcases/test_query.py

```diff
@@ -1618,6 +1618,100 @@ def test_query_using_all_types_of_default_value(self):
         assert res[ct.default_bool_field_name] is False
         assert res[ct.default_string_field_name] == "abc"
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_iterator_normal(self):
```

Comment:
Here is all valid cases, is the invalid cases (like params check test case) in your next pr?